### DISCUSSION TITLE
Whole Home VPN Gateway

### DISCUSSION THREAD

**Discussion by timeshadowrider** [qdm12/gluetun#658] [Q&A]
Is there anyway to turn this into a gateway for my whole home network. My router does not support running a VPN and its great I can connect other Docker Containers to this but I would like to run xbox through this as well and I cant do that with just the proxy gateway.

**Comment by qdm12** [comment]
Read/Subscribe/Contribute to #381 

For my part, I guess it's possible but I'm not sure how. It might require some fiddling on the host/your router which is a bit out of scope, but let me know and I can add a Wiki page for everyone to use 😉 

EDIT changed 581 to #381
